open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/OMRSMetadataCollection.java

```diff
@@ -814,6 +839,26 @@ public abstract EntityDetail getEntityDetail(String     userId,
                                                                          EntityProxyOnlyException,
                                                                          UserNotAuthorizedException;
 
+    /**
+     * Return the header, classifications and properties of a specific entity proxy.  This requires the entity
+     * object to be available.
+     *
+     * @param userId unique identifier for requesting user.
+     * @param guid String unique identifier for the entity.
+     *
+     * @return EntityProxy structure.
+     *
+     * @throws InvalidParameterException the guid is null.
+     * @throws RepositoryErrorException there is a problem communicating with the metadata repository where
+     *                                 the metadata collection is stored.
+     * @throws EntityNotKnownException the requested entity instance is not known in the metadata collection.
+     * @throws UserNotAuthorizedException the userId is not permitted to perform this operation.
+     */
+    public abstract EntityProxy getEntityProxy(String     userId,
```

Comment:
Any method added to OMRSMetadataCollection can not be abstract - it must be implemented for backward compatibility.